web/ui/react-app/src/pages/targets/Filter.module.css

```diff
@@ -0,0 +1,5 @@
+.btn {
+  border-radius: 0px !important;
+  margin-top: 0.6em;
+  margin-bottom: 15px;
+}
```

Comment:
Btw. I don't think we should keep the styling of the old UI's buttons verbatim. Let's just use the standard Bootstrap style (rounded corners and all)?

Reply:
sure, sounds good